pkg/apis/kubeone/validation/validation.go

```diff
@@ -42,7 +42,7 @@ var (
 	// minVersionConstraint defines the minimum Kubernetes version supported by KubeOne
 	minVersionConstraint = semverutil.MustParseConstraint(">= 1.29")
```

Comment:
Let's bump this to 1.30, this can be done in a follow up as well.

Reply:
:heavy_check_mark: